tests/conftest.py

```diff
@@ -266,12 +268,19 @@ def small_array_filters(small_array: np.ndarray, load_scan: LoadScan, filter_con
 # IO fixtures
 @pytest.fixture
 def load_scan(loading_config: dict) -> LoadScan:
-    """Instantiate a LoadScan object."""
+    """Instantiate a LoadScan object from a .spm file."""
     scan_loader = LoadScan(RESOURCES / "minicircle.spm", **loading_config)
     return scan_loader
 
 
 @pytest.fixture
+def load_scan_ibw() -> LoadScan:
+    """Instantiate a LoadScan object from a .ibw file."""
+    scan_loader = LoadScan(RESOURCES / "minicircle2.ibw", channel = "HeightTracee")
```

Comment:
Is this a typo?

```suggestion
    scan_loader = LoadScan(RESOURCES / "minicircle2.ibw", channel = "HeightTrace")
```

Reply:
Yes but sadly not on our side. Seems like the default forward height trace channel is named `HeightTracee`. You can check the list of possible channels for the `ibw` file is you have a channel that doesn't exist in the config, e.g. "hsjkbslbv" or "HeightTrace"

Reply:
Perhaps worth raising an issue (or even forking, correcting and submitting a PR) in the packages repository?

Reply:
I think it might be from the instrument as that is where the file is saved